### PR TITLE
bug fix for deployments flow

### PR DESCRIPTION
**Pull request by supritat**
### Requirements

* Filling out the template is required. Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.
* All new code requires tests to ensure against regressions

### CODE REVIEW

File: jenkins-build-pack-lambda/Jenkinsfile [hunk 1]
```diff
@@ -360,20 +366,22 @@ def buildLambda(String runtime) {
 		sh "pip install -r requirements.txt -t library"
 		sh "touch library/__init__.py"
 		// create virtual environment and install pytest
-		sh """
+		sh ""
+		"
 		pip install virtualenv
 		virtualenv venv
-		. venv/bin/activate
+			.venv / bin / activate
```
Comment (suryajak) [inline jenkins-build-pack-lambda/Jenkinsfile]: revert this change

Comment (suryajak) [reply jenkins-build-pack-lambda/Jenkinsfile]: Did the IDE or some extension do this for you? Sini also had a similar change on her PR and am wondering what is triggering this? What IDE and extensions are you using for indentation/formatting?

File: jenkins-build-pack-lambda/Jenkinsfile [hunk 2]
```diff
@@ -763,27 +772,27 @@ def generateAssetMap(provider, providerId, type, created_by) {
 @NonCPS
 def parseJson(jsonString) {
 	def lazyMap = new groovy.json.JsonSlurperClassic().parseText(jsonString)
-	def m = [:]
+	def m = [: ]
```
Comment (suryajak) [inline jenkins-build-pack-lambda/Jenkinsfile]: remove space here

File: jenkins-build-pack-lambda/Jenkinsfile (hunk 1 above)
Comment (supritat) [reply jenkins-build-pack-lambda/Jenkinsfile]: Using extension (https://marketplace.visualstudio.com/items?itemName=HookyQR.beautify) for indentation fix.

Comment (supritat) [reply jenkins-build-pack-lambda/Jenkinsfile]: Reverted the changes

File: jenkins-build-pack-lambda/Jenkinsfile (hunk 2 above)
Comment (supritat) [reply jenkins-build-pack-lambda/Jenkinsfile]: removed space
